students/jaewon/users/views.py

```diff
@@ -49,9 +49,13 @@ def post(self, request):
         try:
             email = data["email"]
             password = data["password"]
-            if not Account.objects.filter(email=email, password=password).exists():
+            user = Account.objects.get(email=email)
```

Comment:
line 52 53 한 칸 띄워주세요!
```suggestion
            user = Account.objects.get(email=email)
            
```

Reply:
수정하였습니다.